index.bs

```diff
@@ -1099,16 +1099,16 @@ them as <dfn>roaming authenticators</dfn>.
 
 This distinction is important because there are use-cases where only [=platform authenticators=] are acceptable to a
 [=[RP]=], and conversely ones where only [=roaming authenticators=] are employed. As a concrete example of the former, a
-credential on a [=platform authenticator=] may be used by [=[RPS]=] to quickly and conveniently reauthenticate the user with
+credential on a [=platform authenticator=] MAY be used by [=[RP]=] to quickly and conveniently reauthenticate the user with
 a minimum of friction, e.g., the user will not have to dig around in their pocket for their key fob or phone. As a concrete
-example of the latter, when the user is accessing the [=[RP]=] from a given client for the first time, they may be required to
+example of the latter, when the user is accessing the [=[RP]=] from a given client for the first time, they MAY be asked to
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
changed to "they may be asked to"